website/static/js/pages/component-settings-page.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+var $osf = require('js/osfHelpers.js');
+var getConfirmationCode = require('js/projectSettings.js').getConfirmationCode;
```

Comment:
Nice. Where do you suggest I add the new file? Can I just add it to osfHelpers? 
Funny enough, the code in projectSettings (getting 5 contributors for display in delete modal) that errors is not used at all though I understand the purpose. That request is not needed since we can pass component/project contributors to getConfirmationCode.